api/genomics/wdl/RandomizeVcf/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# To build and deploy, from this directory:
```

Comment:
We can probably get rid of most of these comments as they're related to Circle

Reply:
👍